lib/ansiblelint/cli.py

```diff
@@ -96,8 +96,8 @@ def get_cli_parser() -> argparse.ArgumentParser:
 
     parser.add_argument('-L', dest='listrules', default=False,
                         action='store_true', help="list all the rules")
-    parser.add_argument('-f', dest='format', default='plain',
-                        choices=['plain', 'rst'],
+    parser.add_argument('-f', dest='format', default='rich',
```

Comment:
I'm wary of changing defaults without some sort of "shout it from the rooftop" behavior. Is there a compelling reason to make this part of a z-stream release, or are we expecting it to go with at least a minor version bump?

Reply:
This does not count as a breaking chance as the format was not parseable. Basically changing format, adding colors is not much different than rephrasing the messages.

Another way to put it: if it does not modify existing tests, it does not need a major bump. Still, if others disagree, I can assure that the next version number will 4.4.0. Just add `major` label to the PR if you think is needed.